introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is what? My name is, who? My name is jikka jikka jikka Slim Shady.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.